pkg/storegateway/batch_series_test.go

```diff
@@ -846,3 +1084,35 @@ func readAllSeriesLabels(it storepb.SeriesSet) []labels.Labels {
 	}
 	return out
 }
+
+// generateSeriesChunks generates seriesEntries with chunks. Each chunk is a random byte slice.
+func generateSeriesChunks(t *testing.T, numSeries int) []seriesEntry {
```

Comment:
[nit] At the cost of being more verbose, I would name it with the name of the data type it returns.

```suggestion
func generateSeriesEntriesWithChunks(t *testing.T, numSeries int) []seriesEntry {
```

Reply:
i hoped that we rename or replace this type with one called `seriesChunks`. But since we're not there yet, I will rename this now

Reply:
Oh yeah me too, but we're not there yet :(